### ISSUE TITLE
Commands for MuJoCo installation

### ISSUE THREAD

**Issue by DanielTakeshi** [openai/evolution-strategies-starter#10]
For anyone who has gotten the code to run successfully, what commands did you put in for your MuJoCo installation? In the dependency file we have this area:

```
#######################################################
# WRITE CODE HERE TO PLACE MUJOCO 1.31 in /opt/mujoco #
# The key file should be in /opt/mujoco/mjkey.txt     #
# Mujoco should be installed in /opt/mujoco/mjpro131  #
#######################################################
```

I assumed it would be something like:

```
sudo cp -r [?????]/mjpro131/ /opt/mujoco/
sudo cp [?????]/mjkey.txt /opt/mujoco/
```

But I am not sure what to put in the `[?????]`. I have MuJoCo 1.31 for my personal computers, but if I try to copy it there with, e.g. `sudo cp -r ~/mjpro131/ /opt/mujoco/` and `sudo cp ~/mjpro131/mjkey.txt /opt/mujoco/` I get the following after building the packer script:

```
    amazon-ebs: + sudo mkdir -p /opt/mujoco
    amazon-ebs: + sudo cp -r /home/ubuntu/mjpro131/ /opt/mujoco/
    amazon-ebs: cp: cannot stat '/home/ubuntu/mjpro131/': No such file or directory
    amazon-ebs: + sudo cp -r /home/ubuntu/mjpro131/mjkey.txt /opt/mujoco/
    amazon-ebs: cp: cannot stat '/home/ubuntu/mjpro131/mjkey.txt': No such file or directory
    amazon-ebs: + sudo tee /etc/profile.d/mujoco.sh
    amazon-ebs: + sudo echo 'export MUJOCO_PY_MJKEY_PATH=/opt/mujoco/mjkey.txt'
    amazon-ebs: + sudo tee -a /etc/profile.d/mujoco.sh
    amazon-ebs: + sudo echo 'export MUJOCO_PY_MJPRO_PATH=/opt/mujoco/mjpro131'
    amazon-ebs: + . /etc/profile.d/mujoco.sh
    amazon-ebs: ++ export MUJOCO_PY_MJKEY_PATH=/opt/mujoco/mjkey.txt
    amazon-ebs: ++ MUJOCO_PY_MJKEY_PATH=/opt/mujoco/mjkey.txt
    amazon-ebs: ++ export MUJOCO_PY_MJPRO_PATH=/opt/mujoco/mjpro131
    amazon-ebs: ++ MUJOCO_PY_MJPRO_PATH=/opt/mujoco/mjpro131
```

In other words, the packer script thinks MuJoCo is in `/home/ubuntu` instead of my personal machine. Is there a different command to use for this?

**Comment by DanielTakeshi** [comment]
Never mind, figured it out. Use the file provisioner before running the script to get the MuJoCo files copied over to the machines.
